hyperspy/_signals/complex_signal.py

```diff
@@ -123,7 +114,8 @@ def change_dtype(self, dtype):
             raise AttributeError(
                 'Complex data can only be converted into other complex dtypes!')
 
-    def angle(self, deg=False):
+    @format_title('angle')
+    def angle(self, angle, deg=False):
```

Comment:
I think that it is preferable not to use a decorator for the methods that take args because it breaks IPython's keyword autocompletion.